decompiler/util/data_decompile.cpp

```diff
@@ -353,16 +353,20 @@ goos::Object decomp_ref_to_inline_array_guess_size(
   int end_label_idx =
       index_of_closest_following_label_in_segment(start_label.offset, my_seg, labels);
 
+  int end_offset = all_words.at(my_seg).size() * 4;
   if (end_label_idx < 0) {
-    throw std::runtime_error(
+    lg::warn(
```

Comment:
feel free to remove this warning if you want

Reply:
i figured id keep it, i feel like this will probably be correct in most cases -- but its rare enough that the warning shouldn't bother us, and in the event it is wrong somewhere, its more obvious